modules/java/CMakeLists.txt

```diff
@@ -485,13 +485,25 @@ if(ANDROID)
           ARCHIVE DESTINATION ${OPENCV_LIB_INSTALL_PATH} COMPONENT java)
 else()
   if(NOT INSTALL_CREATE_DISTRIB)
-    ocv_install_target(${the_module} OPTIONAL EXPORT OpenCVModules
```

Comment:
I believe we should drop `EXPORT OpenCVModules` for "bindings" by default (it is really confusing).
But we need to test Android build more carefully (they like to link with libopencv_java.so).